### DISCUSSION TITLE
How to create the first account?

### DISCUSSION THREAD

**Discussion by romu70** [fr0tt/benotes#90] [Q&A]
Hi,
Pretty all the question is in the title. I've managed to install BeNotes, I'm facing the login screen, and nothing seems to be available to create a user account.

How do I do this?
Thanks.

**Comment by fr0tt** [comment]
Hi,

depending on how you installed it the answer may vary. You can have a look at the [documentation](https://benotes.org/docs/) to find out more. To put it simply, you create your first user with a command, described in the documentation in detail.

**Comment by romu70** [comment]
Thanks.I chose another way to store my notes.Best regards.Le 6 oct. 2023 à 16:26, fr0tt ***@***.***> a écrit :﻿
Hi,
depending on how you installed it the answer may vary. You can have a look at the documentation to find out more. To put it simply, you create your first user with a command, described in the documentation in detail.
